app/src/test/kotlin/compiler/intermediate/FunctionControlFlowPlannerTest.kt

```diff
@@ -113,21 +113,22 @@ class FunctionControlFlowTest {
         }
     }
 
-    private fun test(program: Program) = ControlFlow.createGraphForEachFunction(
+    private val testDiagnostics = object : Diagnostics {
+        override fun report(diagnostic: Diagnostic) {
+            diagnostics.add(diagnostic)
+        }
+
+        override fun hasAnyErrors(): Boolean {
+            throw RuntimeException("This method shouldn't be called")
```

Comment:
Maybe NotImplementedError instead?

Reply:
I'm not the author of this code, but sure.